wordpress-mobile/WordPress-Android
# Issue/10651 material notifications list and settings

Part of #10651

This PR moves notifications list and settings to the material theme and adds a Dark Mode support.

For unread notifications, we are using the same bg color as for multi-selected sites in Site Picker.

![comparison_notification_list](https://user-images.githubusercontent.com/728822/73431146-8f309280-42f4-11ea-9700-9d69500cbd90.png)

To test:
- Navigate to the Notifications tab and make sure that they look ok in both light and dark themes.
- Navigate to Notification settings and confirm that they look and behave as expected in both light and dark themes.

PR submission checklist:
